docs/tutorials/upgrading-ev3dev-from-command-line.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Upgrading Ev3dev from the Command Line
+subject: System
+---
+
+{% include icon.html type="warning" %}
+By upgrading ev3dev you are also upgrading the drivers, so there may be changes that break user software. 
+You should check the ev3dev [news feed](http://www.ev3dev.org/news/){: .alert-link } for breaking changes before upgrading.
+{: .alert .alert-warning}
+
+When you flash an SD card with an ev3dev image, you are "installing" the version of ev3dev that was available at the time that the image was made. However, sometimes you will want to upgrade the version of ev3dev that you are using so that you can take advantage of new features that have been added or bug fixes that have been made. One option to get a new version is to re-flash your SD card with a newer image, but this also deletes any files you have saved on it. Alternatively, you can upgrade the ev3dev system (the device drivers and kernel) without having to delete any of your work. In this tutorial, we will show you how to upgrade ev3dev from the command line.
+
+**Make sure that you have already configured a network connection and connected to ev3dev via SSH before beginning.**
+
+
+Before starting the upgrade process, you will need to have ev3dev download a list of the most recent versions of available software. To do this, run the following command:
+
+
+    sudo apt-get update
+
+
+This might take up to ten minutes, depending on hardware and network connection speed.
+
+------
+
+You have multiple options in how you upgrade ev3dev. You can either upgrade just the ev3dev core (which is the quickest option) or upgrade all software installed on the EV3 (which may take an hour or longer).
```

Comment:
What is the "ev3dev core"? the kernel? What about upgrading other packages one at a time?